sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -695,6 +695,54 @@ inline pi_result ur2piUSMAllocInfoValue(ur_usm_alloc_info_t ParamName,
   return PI_SUCCESS;
 }
 
+// Translate UR program build info values to PI info values
+inline pi_result ur2piProgramBuildInfoValue(ur_program_build_info_t ParamName,
+                                            size_t ParamValueSizePI,
+                                            size_t *ParamValueSizeUR,
+                                            void *ParamValue) {
+  ConvertHelper Value(ParamValueSizePI, ParamValue, ParamValueSizeUR);
+
+  if (ParamName == UR_PROGRAM_BUILD_INFO_BINARY_TYPE) {
+    auto ConvertFunc = [](ur_program_binary_type_t UrValue) {
+      switch (UrValue) {
+      case UR_PROGRAM_BINARY_TYPE_NONE:
+        return PI_PROGRAM_BINARY_TYPE_NONE;
+      case UR_PROGRAM_BINARY_TYPE_COMPILED_OBJECT:
+        return PI_PROGRAM_BINARY_TYPE_COMPILED_OBJECT;
+      case UR_PROGRAM_BINARY_TYPE_LIBRARY:
+        return PI_PROGRAM_BINARY_TYPE_LIBRARY;
+      case UR_PROGRAM_BINARY_TYPE_EXECUTABLE:
+        return PI_PROGRAM_BINARY_TYPE_LIBRARY;

```

Comment:
this should be PI_PROGRAM_BINARY_TYPE_EXECUTABLE, right?

Reply:
👍